Header Header

float64 starterConsistency

float64 timeStamp
float64 enderConsistency
float64 counter
float64 checkSum

uint16 ITCinfo
bool HWfail
bool SGUFail
float64 horizontalMisalign
float64 absorbBlind
float64 distortBlind
uint8 messageCounter
uint8 messageCRC
